sources/Renderer/Bootstrap/FieldRenderer/BsSelectObjectFieldRenderer.php

```diff
@@ -441,7 +448,7 @@ protected function RenderRegularSearch(RenderingOutput &$oOutput)
 						'selector': '.modal[data-source-element="{$sSearchButtonId}"]:first'
 					};
 				}
-				CombodoModal.OpenModal(oOptions);
+				CombodoPortalToolbox.OpenModal(oOptions);
```

Comment:
```suggestion
				CombodoModal.OpenModal(oOptions);
```
Revert this, it is not related to that PR and this API has been deprecated.